india_compliance/gst_india/doctype/turnover_record/turnover_record.py

```
@@ -0,0 +1,83 @@
+    try:
+        if existing:
+            frappe.db.set_value("Turnover Record", existing, "amount", amount)
+        else:
+            doc = frappe.new_doc("Turnover Record")
+            doc.from_date = from_date
+            doc.to_date = to_date
+            doc.gstin = gstin
+            doc.gst_state = gst_state
+            doc.amount = amount
+            doc.insert(ignore_permissions=True)
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)